controlplane/pkg/model/model.go

```diff
@@ -103,7 +103,17 @@ func (i *impl) AddDataplane(dataplane *Dataplane) {
 	logrus.Infof("Dataplane added: %v", dataplane)
 }
 
-func (i *impl) DeleteDataplane(name string) {}
+func (i *impl) DeleteDataplane(name string) {
+	i.Lock()
+	defer i.Unlock()
+
+	for idx, dp := range i.dataplanes {
```

Comment:
Why not store dataplanes in a map rather than a list so that deleting them is easier?

Reply:
Changed to work as maps, for both dataplanes and endpoints.

Reply:
@haiodo Woot!